CHANGELOG.rst

```diff
@@ -33,6 +33,8 @@ in development
 * Add new ``st2garbagecollector`` service which periodically deletes old data from the database
   as configured in the config. By default, no old data is deleted unless explicitly configured in
   the config.
+* All published variables are now available in the result of ActionChain execution under the
```

Comment:
It's worth noting that this slightly "breaks" the "unified" action chain and mistral result format.

IIRC, in Mistral result object we added Mistral specific attributes under `extra` attribute, maybe we should do a similar thing here and keep it consistent.